frontend/javascripts/bundle.js

```diff
@@ -1,27 +1,29 @@
 import React from 'react'
-import ReactDOM from 'react-dom'
-import { Router, Route, IndexRoute, browserHistory } from 'react-router'
 import { Provider } from 'react-redux'
+import ReactDOM from 'react-dom'
+import { Route, BrowserRouter, Switch } from 'react-router-dom'
```

Comment:
__[ESLint]__ <sub>posted by Sider</sub>

Error - 'BrowserRouter' is defined but never used. (no-unused-vars)